README.rst

```diff
@@ -156,6 +156,13 @@ Bug fix release.
 - Require Mopidy < 2 as Mopidy 2.0 breaks the audio API with the upgrade to
   GStreamer 1.
 
+- Use the new Web API for search. Searching through libspotify has been
+  discontinued and is not working anymore. (Fixes: #89)
+
+- Emit a warning if `search_album_count`, `search_artist_count` or
```

Comment:
Should be double backticks. One of the more subtle differences between Markdown and rST.